chain/types/ethtypes/eth_types.go

```diff
@@ -1278,3 +1278,73 @@ type EthTxReceipt struct {
 	Logs              []EthLog    `json:"logs"`
 	Type              EthUint64   `json:"type"`
 }
+
+const errorFunctionSelector = "\x08\xc3\x79\xa0" // Error(string)
+const panicFunctionSelector = "\x4e\x48\x7b\x71" // Panic(uint256)
+
+// panicErrorCodes maps Solidity panic codes to human-readable descriptions.
+var panicErrorCodes = map[uint64]string{
+	0x00: "Panic()",
+	0x01: "Assert()",
+	0x11: "ArithmeticOverflow()",
```

Comment:
this can also be Underflow, unfortunately. Perhaps `ArithmeticOverflowOrUnderflow()` would prevent confusion